lua/astrocommunity/colorscheme/catppuccin/init.lua

```diff
@@ -3,33 +3,13 @@ return {
   name = "catppuccin",
   lazy = true,
   ---@type CatppuccinOptions
+  ---@diagnostic disable: missing-fields
   opts = {
+    auto_integrations = true,
     integrations = {
-      aerial = true,
-      alpha = true,
-      blink_cmp = true,
-      cmp = true,
-      dap = true,
-      dap_ui = true,
-      gitsigns = true,
-      illuminate = true,
-      indent_blankline = true,
-      markdown = true,
-      mason = true,
-      native_lsp = { enabled = true },
-      neotree = true,
-      notify = true,
-      semantic_tokens = true,
-      symbols_outline = true,
-      telescope = true,
-      treesitter = true,
       ts_rainbow = false,
```

Comment:
I wonder why this is off

Reply:
tbh I dont remember. We could try and remove it lol and see if the world explodes.